build.sh

```diff
@@ -17,6 +17,7 @@ print_help_and_exit () {
 
 List of options:
   --release | -r                    : build in \"Release\" mode (default is \"Debug\")
+  --skip_polygeist <polygeist-path> : skip building POLYGEIST
```

Comment:
Use `skip-polygeist` instead of `skip_polygeist` to be consisten with other options.